documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,80 @@
+# **Metadata Forms**
+
+**Concepts**
+
+Metadata forms allow users to add structured contextual information to various entities in the Data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
+
+Metadata forms serve several key purposes:
+
+- Improve data discovery by enabling more consistent, complete, and meaningful metadata
+- Capture domain-specific or organizational metadata standards
+- Enforce governance policies around data documentation
+- Facilitate data lineage and impact analysis
+- Streamline metadata management workflows
+
+![metadata form levels](pictures/metadata_forms/mf_levels.jpg#zoom#shadow)
+
+This user guide provides instructions on how to create, manage, and use metadata forms in data.all. It covers form creation, attaching forms to entities, filling out forms, managing form visibility and permissions, and enforcing form usage. Whether you are a data producer adding metadata, a data consumer leveraging it, or an administrator standardizing on formats, this guide will help you get the most value out of metadata forms.
+
+With customizable forms that can be attached across many entities, metadata forms provide a flexible and extensible mechanism for metadata management. Use this guide to learn how to configure and apply forms to address your specific metadata use cases and data governance needs.
+
```

Comment:
The paragraphs between line 17 and 20 do not provide much extra info. Instead of more "introductory" paragraphs I would explain the different levels of metadata forms. As a new user I have some questions: What does it mean that it is global-wide, does global-wide metadata forms be attached to a dataset? to a share item? what is a share item metadata form?

Reply:
We could have an intro paragraph outlining the lifecycle of a metadata form:
- metadata form is created 
- owners manage permissions?
- once created, the owners? can attach it to entities - once attach it cannot be detached right?
- metadata forms can be edited - how does this affect existing attachments? 

"For a detail explanation of each of these steps, keep reading the subsections below"